### PR TITLE
[helm-docs] Add helm-docs CI

### PR DESCRIPTION
**Pull request by angelnu**
**Description of the change**

Adds a workflow to create a PR with helm-docs updates

**Benefits**

Ensure documentation stays updated

**Possible drawbacks**

More work to integrate changes when submitter did not update documentation on his own fork prior to PR. This happens when submitter forgets to enable actions on his fork.

**Applicable issues**

- NA

**Additional information**

**Checklist** 
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [ ] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [ ] (optional) Variables are documented in the README.md

### CODE REVIEW

**Comment by angelnu** [comment]
Example of PR in my fork - https://github.com/angelnu/charts/pull/5

**Comment by onedr0p** [comment]
@angelnu should we use the script in `hack/gen-helmdocs.sh` to generate the templates? We might want to keep using this wrapper incase things change.

**Comment by angelnu** [comment]
@onedr0p - we could do it but currently the script is ignoring the .helmdocsignore so it adds the templates to all folders.

Once the other PR is merged that .helmdocsignore will not be needed so maybe we merge this PR as it is and once the other PR is merged (including the 8 charts I did not convert yet) we can adopt `hack/gen-helmdocs.sh`

**Comment by onedr0p** [comment]
ok no problem! 

What do you think about if it were to run on a cron instead of when pushes are made?

**Comment by angelnu** [comment]
Yes, we can transform to use an scheduled trigger since we can very quickly process all the charts anyway.

Should we also consider calling in the release CI to ensure we do not build charts with outdated info? Potentially, we could also commit the change so we do not have any PRs to merge.

**Comment by onedr0p** [comment]
We have to wait until we fix the chart releaser action so that it doesn't error out on duplicate tags.

It ties in with my comment on Discord

https://discord.com/channels/673534664354430999/740734177279868949/818477961190965258

**Comment by angelnu** [comment]
Changed to use a daily schedule as trigger.

File: .github/workflows/charts-helm-docs.yaml
Comment (onedr0p) [inline .github/workflows/charts-helm-docs.yaml:41]:
```suggestion
        title: "chore(docs) update helm-docs [ci-skip]"
```

File: .github/workflows/charts-helm-docs.yaml
Comment (onedr0p) [inline .github/workflows/charts-helm-docs.yaml:45]:
```suggestion
        commit-message: "chore(docs) update helm-docs [ci-skip]"
```

**Review by onedr0p — CHANGES_REQUESTED** [review]

**Review by onedr0p — APPROVED** [review]

**Pull request merged**
